core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedClusterMessage.scala

```diff
@@ -94,4 +94,5 @@ private[spark] object CoarseGrainedClusterMessages {
 
   case class KillExecutors(executorIds: Seq[String]) extends CoarseGrainedClusterMessage
 
+  case class PreemptionExecutors(executorIds: Set[String]) extends CoarseGrainedClusterMessage
```

Comment:
+1. In fact, since you're sending a message saying which executors are about to be pre-empted, a better name would be `PreemptedExecutors`. Same thing everywhere else in this patch.